soc/arm/nxp_imx/rt/soc_rt11xx.c

```diff
@@ -492,6 +498,63 @@ void imxrt_audio_codec_pll_init(uint32_t clock_name, uint32_t clk_src,
 }
 #endif
 
+#if CONFIG_MIPI_DSI
+void imxrt_pre_init_display_interface(void)
```

Comment:
note: likely for future, but this is doing clock control work, so there should be a proper clock control driver handling that (unfortunately clock control area is a bit of a mess now)

Reply:
Yes, we will work to improve this and other drivers once we have a more improved clocking infrastructure in place